src/Symfony/Component/Uid/Uuid.php

```diff
@@ -76,10 +76,7 @@ public static function fromString(string $uuid): parent
     {
         // don't use uuid_generate_md5(), some versions are buggy
         $uuid = md5(hex2bin(str_replace('-', '', $namespace->uid)).$name, true);
-        $uuid[8] = $uuid[8] & "\x3F" | "\x80";
-        $uuid = substr_replace(bin2hex($uuid), '-', 8, 0);
-        $uuid = substr_replace($uuid, '-3', 13, 1);
-        $uuid = substr_replace($uuid, '-', 18, 0);
+        $uuid = static::format($uuid, '-3');
 
         return new UuidV3(substr_replace($uuid, '-', 23, 0));
```

Comment:
This `substr_replace` could be moved to `static::format` also

Reply:
ok, thanks